File: C++/Problems/Problems/problem_709.cpp
#include "problem_709.h"

#include <algorithm>

bool problem_709::test() {
    std::string s = "Hello";

    std::string expected = "hello";

    auto result = toLowerCase(s);

    std::cout << result << std::endl;

    return result == expected;
}

std::string problem_709::toLowerCase(std::string s) {
    std::string result;
    for (auto c  : s)
        result += std::tolower(c);
    return result;
}

File: Python/Problems/Problems/problem709.py

from utils import print1DArray, print2DArray
from typing import List, Set
from collections import Counter

from .problem import Problem


class Problem709(Problem):
    def test(self) -> bool:
        s = "Hello"

        expected = "hello"

        result = self.toLowerCase(s)

        print(result)

        return result == expected

    def toLowerCase(self, s: str) -> str:
        return s.lower()